server/api/pipeline_test.go

```diff
@@ -37,6 +37,7 @@ var fakePipeline = &model.Pipeline{
 	ID:     2,
 	Number: 2,
 	Status: model.StatusSuccess,
+	Commit: &model.Commit{},
```

Comment:
why?

Reply:
Tests fail otherwise with nil pointer. This way e.g. `Commit.SHA` is defined as empty string.